dev/storage_base.h

```diff
@@ -328,7 +331,6 @@ namespace sqlite_orm {
                 std::stringstream ss;
                 ss << C::name();
                 auto name = ss.str();
-                ss.flush();
```

Comment:
removing this will flush in the end of the scope -> after `this->create_collation(name, move(func));`. Better to drop unused memory ASAP

Reply:
not new, it is very old but yes it is not done consistently cause I added it only when I remembered about it.
> It's more code for little gain. 
this is how C and C++ work 🙂. It is not so critical cause no one asked for it yet. So it is up to you

Reply:
No worries, I made it state-of-the-art with a later commit in this PR.